src/framework/audio/thirdparty/fluidsynth/fluidsynth-2.3.3/src/synth/fluid_voice.c

```
@@ -2050,3 +2050,10 @@ void fluid_voice_set_custom_filter(fluid_voice_t *voice, enum fluid_iir_filter_t
     UPDATE_RVOICE_GENERIC_I2(fluid_iir_filter_init, &voice->rvoice->resonant_custom_filter, type, flags);
 }
 
+void fluid_voice_set_iir_lowpass_filter_active(fluid_voice_t *voice, int active)
```

Cppcheck (2.20.0)

[style] The function 'fluid_voice_set_iir_lowpass_filter_active' is never used.

(unusedFunction)